kratos/python/add_strategies_to_python.cpp

```diff
@@ -350,6 +350,7 @@ namespace Kratos
             py::class_<ResidualBasedAdjointBossakSchemeType, typename ResidualBasedAdjointBossakSchemeType::Pointer, BaseSchemeType>
             (m, "ResidualBasedAdjointBossakScheme")
             .def(py::init<Kratos::Parameters, AdjointResponseFunction::Pointer>())
+            .def(py::init<const double, AdjointResponseFunction::Pointer>())
```

Comment:
why do you add this ctor if there is already one with `Paramaters`?
This goes a bit opposite of what we usually do and will also not be used once we finally have the factories

Reply:
Its because, we dont have a response function factory, its always application specific, and each response function may have its own parameters.

Reply:
hm I don't understand ...?
Isn't this the same as for all other Kratos classes?

Reply:
removed :)